CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased (0.13.0)]
+### Added
+- A way to obtain where tilesets, templates and maps have been loaded from
```

Comment:
I think your commit title is more informative that your changelog entry. I'd suggest:
```suggestion
- Added a `source` member to `Tileset`, `Map` and `Template`, which stores the resource path they have been loaded from.
```

Reply:
I forgot to finish the changelog message 😅